vcd/resource_vcd_rde_test.go

```diff
@@ -0,0 +1,566 @@
+//go:build rde || ALL || functional
+
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	"regexp"
+	"strings"
+	"testing"
+)
+
+// TestAccVcdRde tests the behaviour of RDE instances.
+func TestAccVcdRde(t *testing.T) {
+	preTestChecks(t)
+	skipIfNotSysAdmin(t)
+
+	var params = StringMap{
+		"FuncName":       t.Name() + "-Step1-and-2",
+		"ProviderSystem": providerVcdSystem,
+		"ProviderOrg1":   providerVcdOrg1,
+		"Nss":            "nss",
+		"Version":        "1.0.0",
+		"Vendor":         "vendor",
+		"Name":           t.Name(),
+		"Resolve":        false,
+		"SchemaPath":     getCurrentDir() + "/../test-resources/rde_type.json",
+		"EntityPath":     getCurrentDir() + "/../test-resources/rde_instance.json",
+		"EntityUrl":      "https://raw.githubusercontent.com/adambarreiro/terraform-provider-vcd/add-rde-support-3/test-resources/rde_instance.json", // FIXME
```

Comment:
Reminder to change this URL to its final destination

Reply:
> Just thinking: is it possible to put it in a GIST?

Yes, but somehow I feel that a Gist would be detached from this repository, in the sense that it would be owned by a user and not by this repository, which is making use of it.

> Maybe extend the bot to capture "githubusercontent.com" and auto-generate these?

Yes, could be useful.

Reply:
> Yes, but somehow I feel that a Gist would be detached from this repository

+1 for this. Let's have it in the repo.